### PR TITLE
EMSUSD-1629 fix copying grouped meshes

### PR DESCRIPTION
**Pull request by pierrebai-adsk**
- Return the correct path of the copied object from the duplicate function.
- Don't try to process the virtual root prim path.
- Prepare the correct export options.
- Add a unit test for duplicate grouped meshes.

### CODE REVIEW

File: plugin/adsk/scripts/mayaUsdMenu.mel [hunk 1]
```diff
@@ -708,7 +708,7 @@ global proc mayaUsdMenu_duplicateToUSD( string $proxy, string $obj )
     }
 
     for ($item in $bulkObjects) {
-        string $exportOptions = `python("import mayaUsdDuplicateAsUsdDataOptions; import mayaUsdOptions; mayaUsdOptions.setAnimateOption('''" + $obj + "''', mayaUsdDuplicateAsUsdDataOptions.getDuplicateAsUsdDataOptionsText())")`;
+        string $exportOptions = `python("import mayaUsdDuplicateAsUsdDataOptions; import mayaUsdOptions; mayaUsdOptions.setAnimateOption('''" + $item + "''', mayaUsdDuplicateAsUsdDataOptions.getDuplicateAsUsdDataOptionsText())")`;
```
Comment (AramAzhari-adsk) [inline plugin/adsk/scripts/mayaUsdMenu.mel:711]: Is obj the parent node? and if so, do we also need to setAnimateOption on that?
Was item ever used before this change?

Comment (pierrebai-adsk) [reply plugin/adsk/scripts/mayaUsdMenu.mel:711]: $item is replacing $obj, but in the original change I forgot to change it here.

**Review by AramAzhari-adsk — APPROVED** [review]
